src/lib/Newsletter.svelte

```diff
@@ -0,0 +1,17 @@
+<div class="wrapper">
+	<a target="_blank" href="https://tinyletter.com/FeltCoop">
```

Comment:
```suggestion
	<a href="https://tinyletter.com/FeltCoop">
```

Looks like we don't use `target="_blank"` anywhere else, I think it's better to leave that to the user to choose.